airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -328,6 +331,9 @@ definitions:
       namespace:
         description: Optional Source-defined namespace. Currently only used by JDBC destinations to determine what schema to write to. Airbyte streams from the same sources should have the same namespace.
         type: string
+      suggested:
```

Comment:
naming nitpick: can we call it `sync_by_default` or `enabled_by_default`? makes the name more self explanatory